Week 3 Assignment 1/BankAccountClient_W3A1_2.java

```diff
@@ -0,0 +1,209 @@
+// a. A BankAccount will contain the account holders name, account number,
+//		the account creation date (string as of now), branch name and balance and Address
+
+// b. A BankAccount can be either SavingsAccount or CurrentAccount or Salaried Account (a special type of savings account)
+// c. Current account holders do not have withdrawal limit on their account balance whereas Savings account holders
+// 		can only withdraw 25000 per day. 
+// d. CurrentAccount holders should have a minimum account balance of Rs 25000. Regular savings account holders should
+//		maintain 10000 rupees whereas Salaried account holders can maintain 0 balance. 
+// e. There should be a provision to update Nominee details but they cannot update account number and name.
+// f. The address should contain FlatNo, Street, City, State and pin code.
+import java.util.Arrays;
```

Comment:
Every class should be in its own Java file, with a separate class for the client.